### DISCUSSION TITLE
Native sdk issues

### DISCUSSION THREAD

**Discussion by Mohammad-Azizi** [paypal/paypal-android#225] [General]
I checked the documentation you mention native sdk also shared screenshots of it but when i integrate it in my Android app it works as web only on first it display paysheet

**Comment by sshropshire** [comment]
Hi @Mohammad-Azizi the Native PayPal Checkout SDK is no longer supported in `v2` of our SDK. The team that owns native checkout made a decision some time ago to sunset the product to focus efforts elsewhere. Please consider our [PayPal Web Payments](https://github.com/paypal/paypal-android/blob/develop/v2_MIGRATION_GUIDE.md#paypal-web-payments) module as an alternative.
